src/test/java/org/elasticsearch/search/highlight/HighlighterSearchTests.java

```diff
@@ -1914,8 +1879,8 @@ public void testPostingsHighlighter() throws Exception {
 
         logger.info("--> searching on _all, highlighting on field1");
         source = searchSource()
-                .query(termQuery("_all", "test"))
-                .highlight(highlight().field("field1").preTags("<xxx>").postTags("</xxx>"));
+                .query(termQuery("field1", "test"))
```

Comment:
thanks for the review Jun I changed this test to leave querying for `_all` and set `require_field_match` to `false` explicitly. Will change again when #11077, doesn't make much sense to take it into account here.